apps/docs/netsim/README.md

```diff
@@ -197,23 +202,23 @@ receiving the message.
 
 ### Shard Cleaning
 
-Sometimes, clients are disconnected without warning, leaving orphaned rows in 
-shared storage.  Since the server doesn't have any special logic to handle this 
+Sometimes, clients are disconnected without warning, leaving orphaned rows in
+shared storage.  Since the server doesn't have any special logic to handle this
 case, clients take turns performing cleanup on the various shard tables themselves.
 
 1. Each client periodically triggers a cleanup subsystem.
-2. When it starts up, the cleanup subsystem should attempt to get a 'cleaning 
+2. When it starts up, the cleanup subsystem should attempt to get a 'cleaning
    lock' on the network, similar to how routers enforce connection limits.  
    Expired cleaning locks can be overwritten.
-3. When a client successfully gets a cleaning lock, it begins checking for 
-   invalid rows in the shard tables, such that the cleanup cascades from one 
+3. When a client successfully gets a cleaning lock, it begins checking for
+   invalid rows in the shard tables, such that the cleanup cascades from one
    table to the next.
   1. First, any heartbeats over a certain age may be deleted.
-  2. The nodes table gets cleaned first.  Any node that hasn't had an entry in 
+  2. The nodes table gets cleaned first.  Any node that hasn't had an entry in
      the heartbeat table in (TIMEOUT) milliseconds can be removed.
-  3. The wires table comes next.  Any wire with an end connected to a missing 
+  3. The wires table comes next.  Any wire with an end connected to a missing
      node is invalid and may be cleaned up.
-  4. The messages table is next. Any message going to a missing node may be 
+  4. The messages table is next. Any message going to a missing node may be
      cleaned up.
-4. When it's done, the client releases the cleaning lock, letting another 
-   client take a turn.
\ No newline at end of file
+4. When it's done, the client releases the cleaning lock, letting another
```

Comment:
No.  Note the indentation; this is item #4 in the root-level list, not item #5 in the sub-list under item #3.

Now that you point this out though, I realize this whole section on shard cleaning is outdated and no longer applies.  I think I'll tear it out and replace it with something more current.